### PR TITLE
Modifies config.yml file based off microservice template

### PR DESCRIPTION
**Pull request by NicoShanstrom**
## What I did:
modifies config.yml file to be formatted more like the microservice config.yml since that one is getting to the CD. Seems like all is passing, but i cannot try to run cd since we should only allow CD from main. FIgured trying this and merging it into main could see if CD starts working. 

## Did this break anything?
- [x] No
- [ ] Yes
    [ ] This broke some stuff
    [ ] This broke everything

## Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ] Super small fix (Corrected a typo, removed a comment, etc.)
- [ ] Skip all the other stuff and briefly explain the fix.

## Requested feedback: 
- [ ] Yes
    I'd like feedback on (specify line #)... 
- [x] No

## Checklist:
- [ ] If this code needs to be tested, all tests are passing
- [x] The code runs locally
- [ ] There are comments where clarification/ organization is needed
- [x] The code is DRY. If it's not, I tried my best
- [x] I reviewed my code before pushing

## Testing Changes
- [x] No Tests have been changed
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened)

## What's next?
merge to main to check if CD is trying here

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,73 +1,90 @@
...
+
+      # Wait for PostgreSQL to be ready
       - run:
           name: Wait for DB
           command: dockerize -wait tcp://localhost:5432 -timeout 1m
+
+      # Database setup
       - run:
           name: Database setup
-          command: bin/rails db:{create,migrate} --trace
```
Comment (Steddy1Love) [inline .circleci/config.yml:40]: I don't think this change was necessary.

Comment (Steddy1Love) [reply .circleci/config.yml:40]: repeating the "db:" goes against DRY.

**Review by Steddy1Love — CHANGES_REQUESTED** [review]
Would like to pick your brain on why line 40 was changed.

**Comment by Steddy1Love** [comment]
Passes previous CI test, will have to wait and see if main passes random CI checks.

File: .circleci/config.yml (hunk 1 above)
Comment (NicoShanstrom) [reply .circleci/config.yml:40]: not sure if i understand where db: is repeated. isnt that just the command for database setup?

**Pull request merged**